### PR TITLE
add demo notebook for creating connector

### PR DESCRIPTION
**Pull request by ylwu-amzn**
### Description
Add demo notebook for creating connector 
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/tutorials/aws/AIConnectorHelper.ipynb [hunk 1]
```diff
@@ -0,0 +1,1069 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5f2363c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```
Comment (Zhangxunmt) [inline docs/tutorials/aws/AIConnectorHelper.ipynb:9]: Are these lines really needed for { "cells": ...} ? They seems not appearing in your own repo?

File: docs/tutorials/aws/AIConnectorHelper.ipynb [hunk 2]
```diff
@@ -0,0 +1,1069 @@
...
+    "import boto3\n",
+    "from botocore.exceptions import BotoCoreError\n",
+    "import json\n",
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "from requests_aws4auth import AWS4Auth\n",
+    "import time\n",
+    "\n",
```
Comment (Zhangxunmt) [inline docs/tutorials/aws/AIConnectorHelper.ipynb:17]: these "\n" are not necessary too?

File: docs/tutorials/aws/AIConnectorHelper.ipynb [hunk 3]
```diff
@@ -0,0 +1,1069 @@
...
+   "source": [
+    "region = '...' # set your AWS region, for example us-west-2\n",
+    "opensearch_domain_name = '...' # set your AWS OpenSearch domain name\n",
+    "opensearch_domain_username = '...' # set your AWS OpenSearch domain admin username\n",
+    "opensearch_domain_password = '...' # set your domain password\n",
+    "\n",
+    "aws_user_name = '...' # set your AWS IAM user name, not IAM user ARN. \n",
+    "            # To avoid permission issue and quick start, you can use user whith AdministratorAccess policy\n",
+    "            # Configure this user's access key and secret key in ~/.aws/credential \n",
+    "\n",
```
Comment (Zhangxunmt) [inline docs/tutorials/aws/AIConnectorHelper.ipynb:523]: Can we provide an example here in ~/.aws/credential just to inform those people who are not familiar with the IAM Role permissions? Like this:
[You IAM Role Name]
aws_access_key_id = YOUR_ACCESS_KEY_ID
aws_secret_access_key = YOUR_SECRET_ACCESS_KEY

File: docs/tutorials/aws/AIConnectorHelper.ipynb (hunk 1 above)
Comment (ylwu-amzn) [reply docs/tutorials/aws/AIConnectorHelper.ipynb:9]: Yes, they need these

File: docs/tutorials/aws/AIConnectorHelper.ipynb (hunk 2 above)
Comment (ylwu-amzn) [reply docs/tutorials/aws/AIConnectorHelper.ipynb:17]: It's necessary to make the code more readable by adding an empty line

File: docs/tutorials/aws/AIConnectorHelper.ipynb (hunk 3 above)
Comment (ylwu-amzn) [reply docs/tutorials/aws/AIConnectorHelper.ipynb]: done

**Review by Zhangxunmt — DISMISSED** [review]

**Comment by b4sjoo** [comment]
LGTM

**Review by b4sjoo — APPROVED** [review]

**Review by Zhangxunmt — APPROVED** [review]

**Pull request merged**